### DISCUSSION TITLE
This localhost page can’t be found No webpage was found for the web address: https://localhost/ HTTP ERROR 404

### DISCUSSION THREAD

**Discussion by amitleuva1987** [dunglas/symfony-docker#360] [General]
I just clone the symfony-docker repository and run below commands.

`docker compose build --pull --no-cache`

`docker compose up`

then, I visited https://localhost and I see below error

This localhost page can’t be found No webpage was found for the web address: https://localhost/ HTTP ERROR 404

What could be the issues?   

Have I missed any step?

**Comment by maxhelias** [comment]
> Have I missed any step?

Yes start coding to have a route/page

**Comment by amitleuva1987** [comment]
@maxhelias , I also don't see symfony setup, how can I setup symfony/skeleton with this repository?

**Reply by maxhelias** [reply]
I recommend you to inform yourself about the project before to use it.
